packages/web-app-files/src/composables/actions/files/useFileActions.ts

```diff
@@ -39,6 +38,9 @@ export const useFileActions = ({ store }: { store?: Store<any> } = {}) => {
   const router = useRouter()
   const { $gettext, interpolate: $gettextInterpolate } = useGettext()
   const isSearchActive = useIsSearchActive()
+  const appProvidersEnabled = computed(() => {
+    return !!unref(useCapabilityFilesAppProviders(store)).find((appProvider) => appProvider.enabled)
```

Comment:
I needed to provide the `store` as param here. Otherwise triggering the default action is broken accessing `getCurrentInstance().proxy` in `useStore`. Didn't check why, but would be interesting to know... @dschmidt any ideas?

Reply:
turns out: `useCapabilityFilesAppProviders` must not be initialized within the `computed`.

Reply:
... composables in general must be initialized while `setup` is executed :)